.editorconfig

```diff
@@ -4,6 +4,7 @@ root = true
 indent_style = space
 indent_size = 2
 tab_width = 2
+end_of_line = lf
```

Comment:
The problem is that currently files have CRLF so changing this rule without converting all files will lead to mixed line endings.